README.md

```diff
@@ -124,32 +128,43 @@ The prop types of the `KeyHandler` component are:
 | ------------ | -------- | ---------- | --------- | ------------------------------------------------ |
 | keyEventName | string   | no         | `'keyup'` | `'keydown'`, `'keypress'` or `'keyup'`           |
 | keyValue     | string   | yes __\*__ |           | Any given [KeyboardEvent.key]                    |
-| keyCode      | number   | yes __\*__ |           | Any given [KeyboardEvent.keyCode]                |
+| code         | string   | yes __\*__ |           | Any given [KeyboardEvent.code]         |
+| keyCode&dagger;     | number   | yes __\*__ |           | Any given [KeyboardEvent.keyCode]      |
 | onKeyHandle  | function | yes        |           | Function that is called when they key is handled |
 
-__\*__ You should pass only one of these two props: `keyValue` or `keyCode`. [Which one do I pick?](#keyvalue-keycode)
+__\*__ You should pass at least one of these props.
+
+&dagger; _Note_ that the keyCode is frequently browser specific and has therefore be set as
+deprecated, see [MDN for details](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)
 
 [Example](demo/components/examples/component/index.js)
 
 ### Form key handling
 
 This library does not handle key events for form elements such as `<input />` and `<textarea />`.
 
-React does a fine job supporting these already via [keyboard events](https://facebook.github.io/react/docs/events.html#keyboard-events).
+React does a fine job supporting these already via [keyboard events](https://reactjs.org/docs/events.html#keyboard-events).
 
 [Examples](demo/components/examples/input/)
 
 ## Key event names
 
-TODO: explain the differences between the different key events.
```

Comment:
Can you leave the above and add a new section for the new documentation you've added: 

```md
## `keyValue'`, `code` and `keyCode`
```

We'll also need to update the TOC because the heading changed